specification/k3po.control/src/main/scripts/org/kaazing/specification/control/accept.error.if.no.prepare.rpt

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+accept tcp://localhost:12345
+accepted
+connected
+
+read "START\n"
+read "\n"
+
+write "ERROR\n"
```

Comment:
IMHO, the driver is used to drive deterministic tests.  An error on the driver or by an invalid command etc always defines that the test is fatally flawed and can not recover, and the first thing to fix is what caused the error.  So I don't see a use case to support recovery when an error occurs.

Secondly, if there was such a use case.  I think it would be important to still have a way to identify a fatal error (don't retry) versus a recoverable error.  By defaulting everything to the fatal error case now, we can always open up/expand to the recoverable error case in the future.  But by defaulting to the recoverable error case now, means we need to add the mechanism for the fatal error case now, when it isn't clear we need to support such use cases yet.